bin/test_common.h

```diff
@@ -115,7 +115,7 @@ void
 pba_init (struct packout_buf_allocator *, unsigned max);
 
 void *
-pba_allocate (void *packout_buf_allocator, void*, unsigned short, char);
+pba_allocate (void *packout_buf_allocator, void*, void *conn_ctx, unsigned short, char);
```

Comment:
This would be `lsquic_conn_get_ctx_t *`

Reply:
i just voided it here so that it compiled, because as of this header the lsquic_conn_get_ctx_t still hasn't been defined. I figured bc it's an example / not written to display this feature that the least friction path was okay.

Reply:
Oh I see.  I take it back, then.